examples/common.h

```diff
@@ -11,7 +11,7 @@
  * with this software. If not, see
  * <http://creativecommons.org/publicdomain/zero/1.0/>.
  */
-
+#define _GNU_SOURCE
```

Comment:
avoid warnings /log.c:195:5: warning: implicit declaration of function ‘strcasestr’ [-Wimplicit-function-declaration]
and for the sake of supporting old versions de glibc.

Info 
http://linux.die.net/man/3/strcasestr
http://www.gnu.org/software/libc/manual/html_node/Feature-Test-Macros.html